docs/1.10/build-customize-contribute/registry-landscape.md

```diff
@@ -1,10 +1,8 @@
-[Back to table of contents](../index.md)
+---
+title: Registry Landscape
+---
 
-----------
-
-# Registry Landscape  
-
-The cloud native ecosystem is moving rapidly–registries and their feature sets are no exception. We've made our best effort to survey the container registry landscape and compare to our core feature set.
+The cloud native ecosystem is moving rapidly—registries and their feature sets are no exception. We've made our best effort to survey the container registry landscape and compare to our core feature set.
```

Comment:
This doesn't read right, seems unnecessary to make this assessment.  @stuclem , can you please double check some of the wording

Reply:
@xaleeks I debated whether or not to include this page in the doc reorg. Do you want to remove it?

Reply:
i think we need the page @stuclem . this is also a requirement from CNCF as part of the graduation process. we can just reword the content up on top

Reply:
OK, thanks @michmike